magento/magento2
# magento/magento2#12163: UPS Shipping not listed in REST

### Description
\Magento\Quote\Model\ShippingMethodManagement::getList now returns all possible shipping methods.

### Fixed Issues (if relevant)
1. magento/magento2#12163: UPS Shipping not listed in REST

### Manual testing scenarios
1. Enable "Flat Rate" (enabled by default) and "Free Shipping" shipping methods.
2. Create a quote, go to the Checkout page.
3. Fill the all required fields, choose any shipping method, press "Next".
4. Now send request via REST API to [your Magento url]/rest/V1/carts/:cartId/shipping-methods
5. Make sure response contains both enabled in step 1 shipping methods.

### Contribution checklist
